cmk/gui/wato/pages/check_catalog.py

```diff
@@ -381,6 +381,12 @@ def _man_page_catalog_topics():
             _("Generic check plugins"),
             _("Plugins for local agent extensions or communication with the agent in general"),
         ),
+        (
+            "virtual",
+            False,
+            _("Virtualization"),
+            _("Monitoring of classic virtual environment like ESX, Nutanix and HyperV"),
```

Comment:
I would like to remove this. All the current ESX checkman entries use the os/[misc,kernel,storage,...] catalog. This name is better. I just want to avoid going through all the existing check manuals and updating them.

Reply:
I have there  a little bit different view on the "real world" IT. These checks have nothing to do with Operating Systems.
In my opinion is the current situation with the sorting of checks for the virtual environment checks very bad.

But this i cannot decide. It's your (tribe29) decision what to use. But i would strongly encourage to make a clear separation between OS checks (Linux/Windows/Unix) and virtualization infrastucture like (VMWare/Proxmox/HyperV/Nutanix).

Reply:
we decided to keep the new virtualization topic.